network/tcp.go

```diff
@@ -277,7 +286,9 @@ func (t *TCPListener) bind(addr Address) error {
 	return nil
 }
 
-// Listen implements the Listener interface
+// Listen starts to listen for incoming connections and calls fn for every
+// connection-request it receives.
+// In case of an error it returns it.
```

Comment:
`The call returns in case of an error` ? Just `it returns it` is not nice looking ;)